syntax = "proto3";

package order_management;

// Servicio gRPC
service OrderManagement {
  // MÃ©todo para crear un pedido
  rpc CreateOrder (OrderRequest) returns (OrderResponse);
}

// Mensaje de solicitud para crear un pedido
message OrderRequest {
  string product_name = 1;
  double price = 2;
  string payment_gateway = 3;
  string card_brand = 4;
  string bank = 5;
  string shipping_address = 6;
  string email = 7;
}

// Mensaje de respuesta para el pedido
message OrderResponse {
  string order_id = 1;
  string status = 2;
}
